applications/ecallmgr/src/ecallmgr_fs_bridge.erl

```diff
@@ -158,15 +158,27 @@ handle_bypass_media(DP, _Node, _UUID, #channel{profile=ChannelProfile}, JObj) ->
     end.
 
 -spec maybe_bypass_endpoint_media(kz_json:objects(), ne_binary(), ne_binary(), kz_proplist()) -> kz_proplist().
-maybe_bypass_endpoint_media([Endpoint], BridgeProfile, ChannelProfile, DP) ->
-    EndpointProfile = kz_json:get_value(<<"SIP-Interface">>, Endpoint, BridgeProfile),
-    case kz_json:is_true(<<"Bypass-Media">>, Endpoint)
-        andalso EndpointProfile =:= ChannelProfile of
+maybe_bypass_endpoint_media(Endpoints, BridgeProfile, ChannelProfile, DP) ->
+    BypassOnAll = lists:foldl(fun(EP, 'none') ->
```

Comment:
`lists:all/2` feels more appropriate to use instead of the fold.

```erlang
ShouldBypass = lists:all(fun(Endpoint) -> is_bypass_endpoint_media_enabled(Endpoint, BridgeProfile, ChannelProfile) end, Endpoints),
case ShouldBypass of
    'true' -> [{...} | DP];
    'false' -> DP
end.
```
Something like that looks a little more concise (but does it work? :) )